sail.py

```diff
@@ -16,8 +16,8 @@ def main():
     args = get_args()
     finalize_batches = args.finalize_batches
 
-    # Create a Sail client to handle making requests to Scale
-    client = Sail(os.environ["API_KEY"])
+    # Create a Python SDK client to handle making requests to Scale
+    client = ScaleClient(os.environ["API_KEY"])
```

Comment:
```suggestion
    client = ScaleClient(os.environ["API_KEY"], "sail")
```

You can add `sail` standalone, or preferably with a version extension like `sail/v1.2` so requests from sail package can be logged within UserAgent parameter.